logging/logging.go

```diff
@@ -700,7 +693,8 @@ func fromHTTPRequest(r *HTTPRequest) *logtypepb.HttpRequest {
 		return nil
 	}
 	if r.Request == nil {
-		panic("HTTPRequest must have a non-nil Request")
+		log.Println("logging: HTTPRequest must have a non-nil Request")
```

Comment:
Should we be adding logging calls here? The existing `log` calls are configurable with `OnError`. Perhaps we should reuse that for the `default` case below? Not sure if there is a good way to plumb it through.